packages/openapi/src/openapi-request-builder.spec.ts

```diff
@@ -206,33 +234,6 @@ describe('openapi-request-builder', () => {
     expect(response.data).toBe('iss token used on the way');
   });
 
-  it('addCustomHeaders', async () => {
```

Comment:
[q] Isn't this still a valid test case?

Reply:
Would the `addCustomHeaders` function still be required after this change? All mandatory headers are set by default as usual and users can add additional headers as normal parameters now.

Reply:
I believe it is.
1. Internally, we still maintain normal request headers (api spec) and custom headers separately.
2. With these changes, the user can still only add request headers in the function that are part of the spec. I don't think the types provide extensibility. Any headers that are still not in the spec, need to be added via `addCustomHeaders` function.

@marikaner

Reply:
yes it is 😊